lib/web/mwi_wizards.go

```diff
@@ -0,0 +1,181 @@
+package web
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gravitational/trace"
+	"github.com/julienschmidt/httprouter"
+
+	headerv1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/header/v1"
+	machineidv1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/machineid/v1"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/tfgen"
+	"github.com/gravitational/teleport/lib/tfgen/transform"
+)
+
+// machineIDWizard generates IaC code for the Machine Identity CI/CD wizards.
+func (h *Handler) machineIDWizard(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *SessionContext) (any, error) {
```

Comment:
I wonder if creating a different handler per wizard may make life easier as we add more in the future - I can see this request body/handler implementation getting pretty gnarly over time.

Reply:
Yeah, I had started with that but I think there'll be a lot of commonality with this first set of CI/CD wizards - where they essentially have a source (GitHub) and a destination (Kubernetes). Happy to make this a GHA+K8s specific endpoint if you prefer though!

Reply:
My only thought is that having multiple distinct endpoints gives us a bit more flexibility - I imagine most of the implementation could be powered by shared helpers etc. But, I've been burnt a lot in the past by severely overloaded endpoints so perhaps I'm overly cautious here. I'll leave it down to you two to decide.